training_record/services.py

```diff
@@ -126,3 +128,16 @@ def create_campus_records_from_excel(file):
                 records.add(record)
 
         return len(records)
+
+
+class CampusEventFeedbackService:
+    '''Provide services for Record.'''
+    @staticmethod
+    def create_feedback(record, feedback):
+        '''Create feedback for campus-event'''
```

Comment:
Please update the comment so that the full business logic (about record will be updated) is clear to its caller.